nixos/modules/services/audio/librespot.nix

```diff
@@ -0,0 +1,205 @@
+{ config, pkgs, lib, ... }:
```

Comment:
Module must be added to [nixos/modules/module-list.nix](https://github.com/NixOS/nixpkgs/blob/43f383c46472f3284d8ebe9b61f779270ffd1f78/nixos/modules/module-list.nix) or else it won't be evaluated.